manifests/profile/hathitrust/ingest_jobs.pp

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::hathitrust::ingest_jobs
+#
+# Manage the ingest systemd service for hathitrust
+#
+# @example
+#   include nebula::profile::hathitrust::ingest_jobs
+class nebula::profile::hathitrust::ingest_jobs(
+  String $config,
+  String $feed_home,
+  Array[String] $dcu_recipients,
+  String $crms_renewals_source,
+  String $crms_renewals_dest,
+  String $recipient,
+  String $rights_db_config,
+  String $rights_log = '/tmp/populate_rights.log'
+) {
+
+  $feed_perl = "perl -I ${feed_home}/lib ${feed_home}/bin"
+  $feed_log  = "${feed_home}/var/log"
+  $joined_dcu_recipients = $dcu_recipients.join(',')
+  $base_env = [
+    "MAILTO=${recipient}",
+    "FEED_HOME=${feed_home}"
+  ]
+
+
+  cron {
+    default:
+      environment => $base_env + [ "HTFEED_CONFIG=${config}" ],
+      user        => 'libadm';
+
+    ## HOURLY  JOBS
+
+    # Pickup rights from CRMS & aleph and populate the rights database
+    'CRMS rights pickup':
+      # TODO: install loadrights script
+      command     => "${feed_perl}/feed.hourly/populate_rights_data.pl --pickup \
+--screen --level INFO 2>&1 >> ${rights_log}",
+      environment => $base_env + [ "HTFEED_CONFIG=${rights_db_config}" ],
+
+      minute      => 25;
+
+    # Queue volumes for conversion on GRIN and check their statuses
+    'GRIN status check':
```

Comment:
This has some side effects beyond checking -- is there a better label for the resource/cron comment?

Reply:
Will change to "GRIN conversion / status check"